### ISSUE TITLE
Setup without RDS?

### ISSUE THREAD

**Issue by joshk0** [mscifo/pressless#4]
If i'm not mistaken, this is serverless _except for the database server_ -- is that true? Do I need to set up a RDS instance for this?

The appeal of the 'serverless wordpress' idea is that, if no one is accessing it, it costs nothing at rest except for S3 persistence. But if I have to pay to keep an RDS instance turned on, that defeats the purpose.

How close is this to being able to run the RDS only while the WordPress instance is being accessed?

Thanks for a pretty cool project so far anyway!

**Comment by mscifo** [comment]
Yes, this is serverless except for the database server.  You don't have to use RDS, you can choose to use an external or self hosted database server.

Wordpress itself has a core dependency on MySQL as a data store.  I did play around with replacing MySQL with SQLite, but that had its own problems.  If you're looking for a 100% serverless blog solution, I'd suggest using a static site generator or something like NetlifyCMS.

If you'd still like to use Wordpress but are looking for a serverless datastore component, waiting for [Aurora Serverless](https://aws.amazon.com/rds/aurora/serverless/) might be your best option, which is currently in preview.

**Comment by cscetbon** [comment]
And now that Aurora Serverless is out. Did anyone played with it ?

**Comment by hello-josh** [comment]
Aurora Serverless should work with no modifications since it is MySQL compatible.
